packages/yourloops/components/alarm/alarms-content-configuration.tsx

```diff
@@ -258,176 +255,177 @@ function AlarmsContentConfiguration(props: AlarmsContentConfigurationProps): JSX
+              <Typography>{t('time-spent-severe-hypoglycemia')}</Typography>
+              <div className={classes.dropdown}>
+                <BasicDropdown
+                  key={`hypo-threshold-${hypoThreshold.value}`}
+                  id="hypo-threshold"
+                  defaultValue={`${hypoThreshold.value}%` ?? ''}
+                  values={PERCENTAGES}
+                  error={hypoThreshold.error}
+                  onSelect={(value) => onBasicDropdownSelect(value, setHypoThreshold)}
+                />
+              </div>
             </div>
+            {!patient &&
+              <Typography className={classes.defaultLabel}>{t('default', { value: '5%' })}</Typography>
+            }
           </div>
-          {!patient &&
-            <Typography className={classes.defaultLabel}>{t('default', { value: '5%' })}</Typography>
-          }
-        </div>
-      </Box>
-
-      <Divider variant="middle" className={classes.divider} />
+        </Box>
+        <Divider variant="middle" className={classes.divider} />
 
-      <Typography className={classes.categoryTitle}>
-        3. {t('data-not-transmitted')}
-      </Typography>
-      <Typography variant="caption" className={classes.categoryInfo}>
-        {t('current-trigger-setting-data', { nonDataThreshold: nonDataTxThreshold.value })}
-      </Typography>
-      <Box display="flex" marginTop={2}>
-        <div className={classes.subCategoryContainer}>
-          <Typography className={classes.subCategoryTitle}>A. {t('event-trigger-threshold')}</Typography>
-          <div className={classes.valueSelection}>
-            <Typography>{t('time-spent-without-uploaded-data')}</Typography>
-            <div className={classes.dropdown}>
-              <BasicDropdown
-                key={`tir-dropdown-${nonDataTxThreshold.value}`}
-                id={'non-data'}
-                defaultValue={`${nonDataTxThreshold.value}%` ?? ''}
-                values={PERCENTAGES.slice(0, 10)}
-                error={nonDataTxThreshold.error}
-                onSelect={(value) => onBasicDropdownSelect(value, setNonDataTxThreshold)}
-              />
+        <Typography className={classes.categoryTitle}>
+          3. {t('data-not-transmitted')}
+        </Typography>
+        <Typography variant="caption" className={classes.categoryInfo}>
+          {t('current-trigger-setting-data', { nonDataThreshold: nonDataTxThreshold.value })}
+        </Typography>
+        <Box display="flex">
+          <div className={classes.subCategoryContainer}>
+            <Typography className={classes.subCategoryTitle}>A. {t('event-trigger-threshold')}</Typography>
+            <div className={classes.valueSelection}>
+              <Typography>{t('time-spent-without-uploaded-data')}</Typography>
+              <div className={classes.dropdown}>
+                <BasicDropdown
+                  key={`tir-dropdown-${nonDataTxThreshold.value}`}
+                  id={'non-data'}
```

Comment:
Curly braces useless when putting a string
```suggestion
                  id="non-data"
```